spec/domain/mailing_lists/bulk_mail/retriever_spec.rb

```diff
@@ -128,121 +253,52 @@
       expect do
         retriever.perform
       end.to change { Message::BulkMail.count }.by(1)
-        .and change { MailLog.count }.by(1)
-        .and change { Delayed::Job.where('handler like "%Messages::DispatchJob%"').count }.by(1)
-        .and change { Delayed::Job.where('handler like "%MailingLists::BulkMail::SenderRejectedMessageJob%"').count }.by(0)
+                                               .and change { MailLog.count }.by(1)
+                                                                            .and change { Delayed::Job.where('handler like "%Messages::DispatchJob%"').count }.by(1)
+                                                                                                                                                              .and change { Delayed::Job.where('handler like "%MailingLists::BulkMail::SenderRejectedMessageJob%"').count }.by(0)
 
       mail_log = MailLog.find_by(mail_hash: 'abcd42')
       expect(mail_log.status).to eq('retrieved')
       expect(mail_log.mail_from).to eq('dude@hitobito.example.com')
 
       message = mail_log.message
-      expect(message.subject).to eq('Mail 42')
+      expect(message.subject).to eq(300.times.map{ "a" }.join(''))
       expect(message.state).to eq('pending')
     end
+  end
+
+  context 'mails with wrong encoded subject' do
```

Comment:
```suggestion
  context 'mail subject' do
```

try to put all your subject related tests in one context